### PR TITLE
Finished Hebrew Translations

### PR DESCRIPTION
**Pull request by tomgs**
Hey @wbaumann,

All done with the Hebrew translations!

A few things I didn't translate:

1. The `preference_defaults.xml` file, since it seems to contain only boolean fields. 
2. Any fields with `translatable="false"` in them.
3. Any mention of the app's name - it doesn't translate well to Hebrew, in my opinion. קבלות חכמות is the literal translation (Ka-ba-lot Kha-kha-mot, phonetically - ["kha" is the very Hebrew-ey "heit" sound](https://en.wikipedia.org/wiki/Heth#Hebrew_%E1%B8%A4et)), which sounds a bit like a marketing scheme by a not-so-clever advertising agency. I think it's best we leave it at that until we have a better name.
4. The `pref_about_about_dialogMessage` field - due to #3 above.
5. The `pref_about_terms_dialogMessage` field (the GNU Affero license) - I think translating messes up the subtle legalese involved, so I left it at that for now. There's probably a good translation somewhere - I just can't seem to find it! 

Cheers,

Tom

### CODE REVIEW

**Comment by tomgs** [comment]
Travis fails for some reason... that's a knows issue?

File: app/src/main/res/values-he/preference_defaults.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    <!-- ============= Preference Distance ================= -->
+    <item name="pref_distance_rate_defaultValue" format="float" type="dimen">0.0</item>
+    <bool name="pref_distance_include_price_in_report_defaultValue">false</bool>
+    <bool name="pref_distance_print_table_defaultValue">true</bool>
+    <bool name="pref_distance_print_daily_defaultValue">false</bool>
+    <bool name="pref_distance_as_price_defaultValue">true</bool>
+
+    <!-- ================= Preference Pro ================== -->
+    <string name="pref_pro_pdf_footer_defaultValue">Report Generated using Smart Receipts for Android</string>
```
Comment (wbaumann) [inline app/src/main/res/values-he/preference_defaults.xml]: This one. But it's okay to stay in English too :)

**Comment by wbaumann** [comment]
And just for reference, I made two minor commits to fix up a few things:

* https://github.com/wbaumann/SmartReceiptsLibrary/commit/9ae7df3db6245c0748781603b00301fe886e91f2
* https://github.com/wbaumann/SmartReceiptsLibrary/commit/7ff2ce3ea5cf0a8946bb9d47d06c28118b2d6399

The second one does the real work (eg replacing `?` with `/?` in Android studio to address some build warnings). Great work though! Thank you!

**Comment by tomgs** [comment]
Sure man, no worries. Was fun - got to see how the translations play out on Android. Quite a nice way of keeping the translations organised and only show what's needed.

**Comment by wbaumann** [comment]
Agreed. Android's management system is pretty solid (or at least easier than iOS in my opinion), but it's good exposure. Looking over your email next :)

**Pull request merged**